pkg/graph/resolvers/trb_request_form_funding_sources_test.go

```diff
@@ -10,21 +10,13 @@ import (
 func (s *ResolverSuite) TestModifyTRBFundingSources() {
 	ctx := s.testConfigs.Context
 
-	anonEua := "ANON"
-
-	trbRequest := models.NewTRBRequest(anonEua)
-	trbRequest.Type = models.TRBTNeedHelp
-	trbRequest.State = models.TRBRequestStateOpen
-	trbRequest, err := CreateTRBRequest(s.testConfigs.Context, models.TRBTBrainstorm, s.testConfigs.Store)
-	s.NoError(err)
+	trbRequest := s.createNewTRBRequest()
 
 	s.Run("create/fetch/update/delete TRB request form funding sources", func() {
-
-		fetched, err := GetFundingSourcesByRequestID(ctx, s.testConfigs.Store, trbRequest.ID)
+		fetched, err := GetTRBFundingSourcesByRequestID(s.ctxWithNewDataloaders(), trbRequest.ID)
 		s.NoError(err)
-		s.NotNil(fetched)
-		// should have no sources initially
-		s.Len(fetched, 0)
+		// current GQL schema returns null instead of an empty array for no funding sources
+		s.Nil(fetched)
```

Comment:
For some reason the previous version of the tests worked with `NotNil`, but the schema dictates a `null` return and that causes the resolver to pass `nil` instead of an empty slice.

Reply:
After local testing to double check this, it appears we're returning an empty slice currently despite the schema saying it's possible we can return `nil`. I went ahead and fixed this test to allow for an empty slice or `nil` here: [a8ecd66](https://github.com/CMS-Enterprise/easi-app/pull/2715/commits/a8ecd66e481e1cfb0911b02ffd16eca1a4aef530)

The front-end is already typed for `null` or `[]`, so actually returning `null` like the schema specifies shouldn't be an issue. I also tested the FE locally to double check this assumption and didn't see any issues.

Reply:
I'd say this might be a good spot to just add a `TODO` comment in the gql schema to make the array non-nullable -- I would expect the schema to define:
```gql
# This
fundingSources: [TRBFundingSource!]!
# Instead of this, which is our current schema:
fundingSources: [TRBFundingSource!]
```